http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -153,6 +154,16 @@ if [[ "$RUNTIME_OPERATION" == 'creating' ]]; then
 
     mkdir -p /work
     mkdir -p /certs
+    # Mount persisent disk
+    export DISK_DEVICE_ID=$(lsblk -o name,serial | grep 'user-disk' | awk '{print $1}')
+    # Only format disk is it hasn't already been formatted
+    if [ "$IS_GCE_FORMATTED" == "false" ] ; then
+      mkfs.ext4 -m 0 -E lazy_itable_init=0,lazy_journal_init=0,discard /dev/${DISK_DEVICE_ID}
```

Comment:
formatting options according to google suggestions: https://cloud.google.com/compute/docs/disks/optimizing-pd-performance#formatting_parameters